airbyte-integrations/connectors/source-github/source_github/spec.json

```diff
@@ -52,6 +55,14 @@
           }
         ]
       },
+      "start_date": {
+        "type": "string",
+        "title": "Start Date",
```

Comment:
Is this optional or required?

Depending on the response - let's put as asterisk or append (Optional) here.

Reply:
required